answers/Tezos.md

```diff
@@ -0,0 +1,70 @@
+
+Holders who delegate their XTZ to bakers have no direct input into decision-making votes, but bakers [may engage](https://twitter.com/JacobArluck/status/1113590733771018245) with their delegators when deciding how to vote.
+
+
+
+### Further reading 
+
+<https://tezos.gitlab.io/mainnet/>
+
+
+
+# Funding
+
+There is only one [implementation](https://gitlab.com/tezos/tezos/) of a Tezos full/baking node.
+
+### How is development funded?
+
+The Tezos Foundation was established in 2017 to hold most of the ICO proceeds (and some XTZ), with a mandate to use these funds to "support to Tezos and related technologies as well as to the Tezos community". The foundation has a [council](https://tezos.foundation/tezos-foundation-council) of 5 people who steer its direction and how it allocates its capital.
+
+Some of the ICO proceeds were [allocated](https://steemit.com/cryptocurrency/@roshania/tezos-ico-transparency-memo) to DLS along with an a portion of the initial XTZ tokens. DLS was created by Arthur Breitman to hold the IP for Tezos protocol software, trademarks, etc. Tezos Foundation and DLS entered into a contract whereby the Foundation would pay 8.5% of the ICO proceeds (around $20 million at the time) to acquire DLS and its assets, then release all of the software under a free MIT license. This deal was included in the ICO terms, with a stipulation that it would be completed once the Tezos mainnet had been in operation and stable for 3 months. The Tezos Foundation and DLS were also granted 76 million XTZ each, with DLS's allocation on a 48 month vesting schedule.
+
+There is very little information available about how the Foundation is allocating its funds. There are [often](https://tezos.foundation/news/tezzigator-receives-grant-to-expand-cloud-hsm-support-for-tezos-bakers) [news](https://tezos.foundation/news/supporting-the-development-of-block-explorers-and-smart-contract-languages-for-the-tezos-ecosystem) [items](https://tezos.foundation/news/tezos-foundation-supports-tezos-riot-forum) on the foundation website that say a particular team/project is being funded, but these have no information about how much funding is being given. 
+
+The foundation is being [audited](https://tezos.foundation/news/statement-on-2017-statutory-audit-completed-by-pwc-switzerland) by PwC but no details of its finances are being openly published. 
```

Comment:
@Hanon's [reply](https://matrix.to/#/!lbzTjhzNbIaDbuAxkS:decred.org/$155791174413926zvbMo:decred.org?via=decred.org&via=matrix.org) in writing channel mentions the Swiss Federal Supervisory Authority. Worth mentioning here?

> s_ben: I wrote this about Tezos: "there is a Tezos Foundation that relies on a Council and the Swiss Federal Supervisory Authority to ensure that the Foundation’s assets are used responsibly and PwC Switzerland to audit the Foundation’s finances and operations. Grants are administered by the Tocqueville Group and all proposals are evaluated by an advisory committee of qualified community members, although the Council has the final vote."